react-app/src/components/UserCard/UserCard.tsx

```diff
@@ -0,0 +1,27 @@
+import { UserFormData } from '../../types/types';
+import './UserCard.scss';
+
+type UserCardProps = {
+  form: UserFormData;
+};
+
+export default function UserCard(props: UserCardProps) {
```

Comment:
use can use prop destructuring too

```suggestion
export default function UserCard({ form: { username, age, email, password, gender, terms, image, country } }: UserCardProps) {
```